src/app.ts

```diff
@@ -47,6 +47,7 @@ app.use(cookieParser());
 app.use(express.static(path.join(scriptDir, 'public/root')));
 app.use(`/manifest.webmanifest`, express.static(path.join(scriptDir, 'public/manifest.webmanifest')));
 app.use(`/robots.txt`, express.static(path.join(scriptDir, 'public/robots.txt')));
+app.use(`/icon.png`, express.static(path.join(scriptDir, 'public/icon.png')));
```

Comment:
You should probably use the png that already exists in the images folder. https://github.com/TriliumNext/Notes/blob/develop/images/app-icons/png/512x512.png

Reply:
Good find. I did a quick `find` and didn't see it, so I generated my own. But this is so much better.

Reply:
Ah I see, this file is only on the `develop` branch and not the last release or `master` branch.